recipes/scalene/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set package_name = "scalene" %}
```

You'll need to change it here too.

```suggestion
{% set name = "scalene" %}
```